src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseAccessService.java

```diff
@@ -76,6 +78,11 @@ public AccessLevel getAccessLevel(ServiceAndUserDetails serviceAndUserDetails) {
         return userRoles;
     }
 
+    public Set<String> getCreateCaseRoles() {
```

Comment:
this should to be renamed to a more generic `getCreateRoles` as it's also returning idam roles (which are not case roles)